pages/common/wp.md

```diff
@@ -0,0 +1,24 @@
+# wp
+
+> The official command-line interface to manage WordPress instances.
+> More information: <https://wp-cli.org/>.
+
+- Describe the current system environment:
```

Comment:
I have never used ws.
what is a current system environment, will a ws beginner know what that is?

Reply:
Someone familiar with WordPress might be able to assume, but it could more explicit if preferred.
It'll display the operating system, and paths/versions of the shell, PHP installation, and WP-CLI installation.

It displays a lot of information, so I'm not sure how to concisely describe it all without being too vague.

> ```sh
> $ wp --info
> OS:	Darwin 16.7.0 Darwin Kernel Version 16.7.0: Thu Jan 11 22:59:40 PST 2018; root:xnu-3789.73.8~1/RELEASE_X86_64 x86_64
> Shell:	/bin/zsh
> PHP binary:    /usr/local/bin/php
> PHP version:    7.0.22
> php.ini used:   /etc/local/etc/php/7.0/php.ini
> WP-CLI root dir:        /home/wp-cli/.wp-cli/vendor/wp-cli/wp-cli
> WP-CLI vendor dir:	    /home/wp-cli/.wp-cli/vendor
> WP-CLI packages dir:    /home/wp-cli/.wp-cli/packages/
> WP-CLI global config:   /home/wp-cli/.wp-cli/config.yml
> WP-CLI project config:
> WP-CLI version: 2.4.0
> ```
> \- https://wp-cli.org/#installing

Might something like this be better? 🤔 

```suggestion
- Describe the operating system, shell, PHP, and WP-CLI installation:
```

Reply:
Thats better IMO

Reply:
I like that better too 👍🏻